### PR TITLE
python3Packages.aiowinreg: 0.0.5 -> 0.0.6

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Update to latest upstream 0.0.6

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 131613` at 3c47d946 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>6 packages built successfully:</summary>
  <ul>
    <li>python38Packages.aiowinreg</li>
    <li>python38Packages.lsassy</li>
    <li>python38Packages.pypykatz</li>
    <li>python39Packages.aiowinreg</li>
    <li>python39Packages.lsassy</li>
    <li>python39Packages.pypykatz</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
@ofborg eval

File: pkgs/development/python-modules/aiowinreg/default.nix [hunk 1]
```diff
@@ -3,23 +3,28 @@
...
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "096663ec3db35fdc7ccc1c2d0d64a11cf64f4baa48955088e42b6a649ce418a5";
+    sha256 = "0h0r9xrz1n8y75f2p21f7phqrlpsymyiipmgzr0lj591irzjmjjy";
   };
 
-  propagatedBuildInputs = [ winacl ];
+  propagatedBuildInputs = [
+    prompt_toolkit
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/aiowinreg/default.nix:21]: in staging is a PR which changes this to prompt-toolkit.

**Comment by siraben** [comment]
Result of `nixpkgs-review pr 131613` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>6 packages built:</summary>
  <ul>
    <li>python38Packages.aiowinreg</li>
    <li>python38Packages.lsassy</li>
    <li>python38Packages.pypykatz</li>
    <li>python39Packages.aiowinreg</li>
    <li>python39Packages.lsassy</li>
    <li>python39Packages.pypykatz</li>
  </ul>
</details>

**Pull request merged**